README.md

```diff
@@ -23,15 +23,15 @@ val = hcat(val, rand(1:2, n_iter, 1, n_chain))
 # construct a Chains object
 chn = Chains(val)
 
-# visualize a density plot / histogram plot, autocorrelation plot and a running average plot
-plot(chn, [:mixeddensity, :autocor, :mean])
+# visualize the MCMC simulation results 
+plot(chn)
```

Comment:
This new default plot looks nice. Do we still have support for the old syntax `plot(chn, [:mixeddensity, :autocor, :mean])`?

Reply:
Unfortunately not. We would need to specify a plot(...) function which would create a naming clash with Plots and to resolve this we would need to import and depend on Plots. This solution has a more clumsy syntax but frees us from Plots and StatPlots.

Reply:
Same here, I'll provide alias functions that support the old syntax. The functions will raise a warning indicating that one should use the new syntax.